### ISSUE TITLE
If multiple non-option arguments are passed, treat them as a command

### ISSUE THREAD

**Issue by dkhalansky** [Miserlou/Loop#33]
`find` and `xargs` allow to pass to them commands in such ways:
`find . -exec ls -l ';'`
`xargs echo 3 4 5`
This is arguably better than `find . -exec "ls -l"` because this way you don't have to think as much about the nested strings that can already be a pain when using the shell.

So it would greatly simplify life in some complex cases if `loop` supported syntax like `loop [options here] -- command arg1 arg`.

**Comment by Miserlou** [comment]
Yep, this'd be a great one to have. I actually didn't know about this syntax.

Dupe of : https://github.com/Miserlou/Loop/issues/16

**Comment by Miserlou** [comment]
If you wanna take a stab at this, here's an example: https://github.com/clap-rs/clap/blob/2c86c8eb4cf3688c75e31332ddbd6942dd62c92b/examples/22_stop_parsing_with_--.rs

**Comment by michaelmior** [comment]
@Miserlou I assume that means you're not opposed to switching to another lib for arg parsing then?

**Comment by michaelmior** [comment]
The challenge seems to be supporting both things like `loop -- a b` and `loop 'a b'` since the straightforward way of supporting the first would be to parse a vector of strings. However, this would mean the second example would give a vector of length 1 with the string `"a b"`.

**Comment by michaelmior** [comment]
Ah, except that shouldn't matter since everything is just passed through to the shell :)
